tck/features/ProcedureCallAcceptance.feature

```diff
@@ -0,0 +1,515 @@
+    Then a SyntaxError should be raised at compile time: InvalidArgumentPassingMode
+
+  Scenario: Standalone call to procedure that takes no arguments
+    And there exists a procedure test.labels() :: (label :: STRING?):
+      | label |
+      | 'A'   |
+      | 'B'   |
+      | 'C'   |
+    When executing query:
+    """
+    CALL test.labels()
+    """
+    Then the result should be, in order:
+      | label |
+      | 'A'   |
+      | 'B'   |
+      | 'C'   |
+    And no side effects
+
+  Scenario: In-query call to procedure that takes no arguments
+    And there exists a procedure test.labels() :: (label :: STRING?):
+      | label |
+      | 'A'   |
+      | 'B'   |
+      | 'C'   |
+    When executing query:
+    """
+    CALL test.labels() YIELD label
+    RETURN label
+    """
+    Then the result should be, in order:
+      | label |
+      | 'A'   |
+      | 'B'   |
+      | 'C'   |
+    And no side effects
+
+  Scenario: Calling the same procedure twice using the same outputs in each call
+    And there exists a procedure test.labels() :: (label :: STRING?):
+      | label |
+      | 'A'   |
+      | 'B'   |
+      | 'C'   |
+    When executing query:
+    """
+    CALL test.labels() YIELD label
+    WITH count(*) AS c
+    CALL test.labels() YIELD label
+    RETURN *
+    """
+    Then the result should be, in order:
+      | label | c |
+      | 'A'   | 3 |
+      | 'B'   | 3 |
+      | 'C'   | 3 |
+    And no side effects
+
+  Scenario: Standalone call to VOID procedure that takes no arguments
+    And there exists a procedure test.doNothing() :: VOID:
+      |
+    When executing query:
+    """
+    CALL test.doNothing()
+    """
+    Then the result should be empty
+    And no side effects
+
+  Scenario: In-query call to VOID procedure that takes no arguments
+    And there exists a procedure test.doNothing() :: VOID:
+      |
+    When executing query:
+    """
+    MATCH (n)
+    CALL test.doNothing()
+    RETURN n
+    """
+    Then the result should be empty
+    And no side effects
+
+  Scenario: In-query call to VOID procedure does not consume rows
+    And there exists a procedure test.doNothing() :: VOID:
+      |
+    And having executed:
+    """
+    CREATE (:A {name: 'a'})
+    CREATE (:B {name: 'b'})
+    CREATE (:C {name: 'c'})
+    """
+    When executing query:
+    """
+    MATCH (n)
+    CALL test.doNothing()
+    RETURN n.name AS `name`
+    """
+    Then the result should be:
+      | name |
+      | 'a'  |
+      | 'b'  |
+      | 'c'  |
+    And no side effects
+
+  Scenario: Standalone call to VOID procedure that takes no arguments, called with implicit arguments
+    And there exists a procedure test.doNothing() :: VOID:
+      |
+    When executing query:
+    """
+    CALL test.doNothing
+    """
+    Then the result should be empty
+    And no side effects
+
+  Scenario: In-query call to procedure that takes no arguments and yields no results
+    And there exists a procedure test.doNothing() :: ():
```

Comment:
This is a procedure that just does not bind any new columns, in the same way as if using `YIELD -`. Weird corner case.